assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -17,50 +17,131 @@ import useLibrary from './useLibrary';
 
 const Image = styled.img`
 	height: 150px;
-	width: 150px;
-	padding: 4px;
-	margin: 4px;
+	width: 48%;
+	padding: 3px;
+	margin: 3px;
 	border: 1px solid white;
 `;
 
+const Title = styled.h3`
+	color: ${ ( { theme } ) => theme.colors.fg.v1 };
+	margin: 0px;
+	font-size: 19px;
+	line-height: 20px;
+	line-height: 1.4em;
+	flex: 2 0 0;
+`;
+
+const Button = styled.button`
+	 background: #242A3B;
```

Comment:
All colors should come from the theming. This is `bg.v3`:

```suggestion
	 background: ${ ( { theme } ) => theme.colors.bg.v3 };
```